README.md

```diff
@@ -683,6 +683,8 @@ Table of Contents
   * [whimsical.com](https://whimsical.com/) - Collaborative flowcharts, wireframes, sticky notes and mind maps. Create up to 4 free boards.
   * [lensdump.com](https://lensdump.com/) - Free cloud image hosting
   * [Adobe XD](https://www.adobe.com/products/xd.html) - Wireframe & Prototyping tool similar to Sketch. Free plan covers: 1 active shared design spec, Adobe Fonts Free (limited set of fonts), 2GB of cloud storage
+  * [Invision App](https://www.invisionapp.com) - UI design and prototyping tool. Desktop and webapp available. Free to use with 1 active prototype.
+  * [Proto.io](https://www.proto.io) - Create fully interactive UI prototypes without coding. Free tier available when free trail ends.
```

Comment:
it does say indeed there is a free tier, but i cant figure out what that means, what features etc? Got a link handy?

Reply:
i believe `trail ends` means `trial ends`

Reply:
@ripienaar Pricing page has an incredibly feint underline... Ironically, not great design 😄 

![image](https://user-images.githubusercontent.com/1271105/69588303-105f6680-0fe0-11ea-95d2-fd8d2f0cb3f5.png)